bash_command_line_101/bash_command_line_101.md

```diff
@@ -2,8 +2,8 @@
 module_id: bash_command_line_101
```

Comment:
This comment refers to a line that isn't in the PR-affected code, so I can't comment there (see https://github.com/isaacs/github/issues/1655).  Ugh.  Anyway, in line 318, consider being a bit more explicit and saying "everything is text.  For example, this is the text we'd write to create a bolded hyperlink:

Reply:
added!